src/main/java/de/flapdoodle/embed/process/distribution/BitSize.java

```diff
@@ -33,7 +33,7 @@ Ben McCann (benmccann@github)
 	public static BitSize detect() {
 		BitSize bitSize = BitSize.B32;
 		String osArch = System.getProperty("os.arch");
-		if (osArch.equals("i686_64") || osArch.equals("x86_64") || osArch.equals("amd64"))
+		if (osArch.equals("i686_64") || osArch.equals("x86_64") || osArch.equals("amd64") || osArch.equals("ppc64le") )
```

Comment:
```suggestion
		if (osArch.equals("i686_64") || osArch.equals("x86_64") || osArch.equals("amd64") || osArch.equals("ppc64le") || osArch.equals("aarch64"))
```

Might as well solve two problems at once :)  This also occurs with AArch64 Java.